.appveyor.yml

```diff
@@ -0,0 +1,27 @@
+environment:
+  matrix:
+    - channel: nightly
+      target: x86_64-pc-windows-msvc
+    - channel: nightly
+      target: i686-pc-windows-msvc
+    - channel: nightly
+      target: x86_64-pc-windows-gnu
+    - channel: nightly
+      target: i686-pc-windows-gnu
```

Comment:
I don't intend to support 32 bit windows, so might as well take this out.

Reply:
Done

Reply:
Thanks, did you see my other comment?

Reply:
Replied to the other comment as well :)